picoquic/packet.c

```diff
@@ -359,7 +359,7 @@ int picoquic_parse_packet_header(
 
          if ((int)length >= 1 + cnxid_length) {
              /* We can identify the connection by its ID */
-             ph->offset = (uint32_t)( 1 + picoquic_parse_connection_id(bytes + 1, cnxid_length, &ph->dest_cnx_id));
+             ph->offset = (uint32_t)((size_t)1 + picoquic_parse_connection_id(bytes + 1, cnxid_length, &ph->dest_cnx_id));
```

Comment:
The final cast to (uint32_t) looks dangerous.

Reply:
Yes, that code dates from before your PR redefining offset as size_t. Will fix.